packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -618,6 +618,101 @@ Expected response body:
 }
 ```
 
+### Client_Structure_MultiClient
+
+- Endpoints:
+  - `get /one`
+  - `get /three`
+  - `get /three`
+  - `get /one`
+  - `get /three`
+  - `get /three`
+
+Include multiple clients in the same spec.
+
+```ts
+const clientA = new ClientAClient();
+const clientB = new ClientBClient();
+
+clientA.renamedOne();
+clientA.renamedThree();
+clientA.renamedFive();
+
+clientB.renamedTwo();
+clientB.renamedFour();
+clientB.renamedSix();
+```
+
+### Client_Structure_RenamedOperation
+
+- Endpoints:
+  - `get /one`
```

Comment:
Is this endpoints too general ? @timotheeguerin

Reply:
pending https://github.com/Azure/cadl-ranch/pull/386 to fix it and will be fixed after rerun the generate-scenarios-summary

Reply:
fixed